### ISSUE TITLE
Backup seed phrase banner is not visible right after account creation

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#10607]
### Description

1. create a new account with https://ci.infra.status.im/job/status-desktop/job/release/job/release%252F0.12.x/4/ build 
2. log in

<img width="1427" alt="Screenshot 2023-05-10 at 10 24 10" src="https://github.com/status-im/status-desktop/assets/82375995/c12f3529-0bfc-4a4a-a084-2edc7fcaf4ba">

**Comment by glitchminer** [comment]
@anastasiyaig , appears OK on Windows and running from release branch on Mac.
![image](https://github.com/status-im/status-desktop/assets/50769329/a2827099-d47b-4137-b866-c9585c07a0dd)

**Comment by caybro** [comment]
I can reproduce with this one (or with any other banner in fact): it happens when you hide (minimize or close to tray) the window while the banner is about to disappear (or just closing down). You are then left with this red glitch or blank space

**Comment by caybro** [comment]
TLDR, the key factor to reproduce is that the window is fully obscured when there's an ongoing animation (either showing up or hiding the banner)

**Comment by alexandraB99** [comment]
@benjthayer we need your input to optimize these banners (besides the backup seed phrase). Currently we get a banner as soon an event triggers which might mean that for example if we get disconnected and connected 2 times in a second, we will show 4 banners in this little time which is not very optimal ux and performance wise. How could we improve/simplify the current banners behavior? Feel free to provide input in this ticket, it will be worked on here.

**Comment by benjthayer** [comment]
> @benjthayer we need your input to optimize these banners (besides the backup seed phrase). Currently we get a banner as soon an event triggers which might mean that for example if we get disconnected and connected 2 times in a second, we will show 4 banners in this little time which is not very optimal ux and performance wise. How could we improve/simplify the current banners behavior? Feel free to provide input in this ticket, it will be worked on here.

My only suggestion here would be for us to include some latency to the banners - i.e. if I go offline, we for an appropriate timeframe (e.g. 3-4 seconds) before reporting it to the user via the banner. We could also check the issue is still in place again just before presenting the banner and if it isn't we don't show it. That way we mitigate against over-communicating to the user and filling their screen with notifications. 

S, in summary, the suggestion would be:

1) Detect issue (don't show notification banner immediately)
2) Wait [n] seconds before showing notification banner (whatever period of time we feel appropriate)
3) Just before showing the notification banner check again to ensure the issue is still valid / persistent
4) Based on the outcome of 3 - show banner if issue persists, show nothing if issue has gone away 

The only downside I see to this from a UX perspective (but perhaps preferable to being flashed lots of notification banners unnecessarily) is that if the issue does persist during the latency window, the app may appear unresponsive in some regards depending on the issue - but it would only be for a couple of seconds.

WDYT @John-44

**Comment by John-44** [comment]
> tion here would be for us to include some latency to the banners - i.e. if I go offline, we for an appropriate timeframe (e.g. 3-4 seconds) before reporting it to the user via the banner. We could also check the issue is still in place again just before presenting the banner and if it isn't we don't show it. That way we mitigate against over-communicating to th



> > @benjthayer we need your input to optimize these banners (besides the backup seed phrase). Currently we get a banner as soon an event triggers which might mean that for example if we get disconnected and connected 2 times in a second, we will show 4 banners in this little time which is not very optimal ux and performance wise. How could we improve/simplify the current banners behavior? Feel free to provide input in this ticket, it will be worked on here.
> 
> My only suggestion here would be for us to include some latency to the banners - i.e. if I go offline, we for an appropriate timeframe (e.g. 3-4 seconds) before reporting it to the user via the banner. We could also check the issue is still in place again just before presenting the banner and if it isn't we don't show it. That way we mitigate against over-communicating to the user and filling their screen with notifications.
> 
> S, in summary, the suggestion would be:
> 
> 1. Detect issue (don't show notification banner immediately)
> 2. Wait [n] seconds before showing notification banner (whatever period of time we feel appropriate)
> 3. Just before showing the notification banner check again to ensure the issue is still valid / persistent
> 4. Based on the outcome of 3 - show banner if issue persists, show nothing if issue has gone away
> 
> The only downside I see to this from a UX perspective (but perhaps preferable to being flashed lots of notification banners unnecessarily) is that if the issue does persist during the latency window, the app may appear unresponsive in some regards depending on the issue - but it would only be for a couple of seconds.
> 
> WDYT @John-44

@benjthayer yes I agree with everything in your message above.

The error banners are currently either:
1. over sensitive, triggered too easily for problems that will resolve themselves a second or two later
2. are displayed as a result of errors that need to be fixed (e.g. the solution to the error banner being displayed is to prevent the error occurring too frequently.

What @benjthayer has suggested is a good solution to 1.  If they don't already exist, there should be issues opened to fix the root cause of error banners being displayed in order to fix 2.

The 'Backup seed phrase banner' isn't an error banner, and should always be visible after account creation.
